ui/selectors/multichain-assets-rates.ts

```diff
@@ -0,0 +1,15 @@
+import { MultichainAssetsRatesControllerState } from '@metamask/assets-controllers';
+
+export type AssetsState = {
+  metamask: MultichainAssetsRatesControllerState;
+};
+
+/**
+ * Gets non-EVM accounts assets rates.
+ *
+ * @param state - Redux state object.
+ * @returns An object containing non-EVM assets per accounts.
+ */
+export function getAssetsRates(state: AssetsState) {
+  return state.metamask.conversionRates;
```

Comment:
Do you think we should move these selectors to ui/selectors/assets.ts instead?

Reply:
I prefer to keep these selectors separate because additional functions, such as retrieving metadata, will be added in the future. Keeping them isolated ensures better maintainability and testability. If we move them to `ui/selectors/assets.ts`, we would also need to relocate the tests, which could reduce modularity and make testing more complex.

Reply:
from a code quality perspective, maintaining a clear separation of concerns helps with readability, maintainability, and scalability.